src/test/java/kitchenpos/bo/MenuBoTest.java

```diff
@@ -0,0 +1,175 @@
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class MenuBoTest {
+
+    @Mock
+    private MenuDao menuDao;
+
+    @Mock
+    private MenuGroupDao menuGroupDao;
+
+    @Mock
+    private MenuProductDao menuProductDao;
+
+    @Mock
+    private ProductDao productDao;
+
+    @InjectMocks
+    private MenuBo menuBo;
+
+    private Menu menu;
+    private MenuProduct menuProduct;
+    private Product product;
+    private MenuGroup menuGroup;
+
+    @BeforeEach
+    void setUp() {
+        final List<MenuProduct> menuProductList = new ArrayList<>();
+
+        menuProduct = new MenuProduct();
+        menuProduct.setMenuId(1L);
+        menuProduct.setProductId(1L);
+        menuProduct.setQuantity(1);
+        menuProductList.add(menuProduct);
+
+        product = new Product();
+        product.setId(1L);
+        product.setName("후라이드");
+        product.setPrice(BigDecimal.valueOf(16000));
+
+        menuGroup = new MenuGroup();
+        menuGroup.setId(1L);
+        menuGroup.setName("두마리메뉴");
+
+        menu = new Menu();
+        menu.setId(1L);
+        menu.setMenuGroupId(1L);
+        menu.setName("후라이드치킨");
+        menu.setPrice(BigDecimal.valueOf(16000));
+        menu.setMenuProducts(menuProductList); ;
+
+        menuBo = new MenuBo(menuDao, menuGroupDao, menuProductDao, productDao);
+    }
+
+    @DisplayName("메뉴를 생성할 수 있다.")
+    @Test
+    void create() {
+        // given
+        given(menuGroupDao.existsById(anyLong())).willReturn(true);
+        given(productDao.findById(anyLong())).willReturn(Optional.of(product));
+        given(menuProductDao.save(any(MenuProduct.class)))
+                .willReturn(menuProduct);
+        given(menuDao.save(any(Menu.class))).willReturn(menu);
+
+        // when
+        final Menu actual = menuBo.create(menu);
+
+        // then
+        assertThat(actual).isNotNull();
+        assertThat(actual.getName()).isEqualTo(menu.getName());
+        assertThat(actual.getPrice()).isEqualTo(menu.getPrice());
+        assertThat(actual.getMenuGroupId()).isEqualTo(menuGroup.getId());
+    }
+
+    @DisplayName("메뉴의 가격은 0보다 커야한다.")
+    @ParameterizedTest
+    @NullSource
+    @ValueSource(strings = {"-1", "-100000"})
+    void priceNull(final BigDecimal price) {
+        // given
+        menu.setPrice(price);
+
+        // then
+        assertThatExceptionOfType(IllegalArgumentException.class)
+                .isThrownBy(() -> menuBo.create(menu));
+    }
+
+    @DisplayName("메뉴는 메뉴 그룹에 존재 해야한다.")
+    @Test
+    void menuGroupException() {
+        // when
+        when(menuGroupDao.existsById(anyLong()))
```

Comment:
`BDDMockito#given(Object)`을 찾아보고 그 기능을 사용해 보면 어떨까요? `Mockito#when(Object)`을 이용한 스터빙(stubbing)은 BDD(behavior-driven development) 스타일에 잘 맞아떨어지지 않아요. 왜냐하면 스터빙은 'When'이 아닌 'Given'에 속하기 때문입니다. 두 메서드의 기능에는 차이는 없습니다. `BDDMockito#given(Object)`의 내부 동작을 살펴보세요. 아래의 글이 도움 되면 좋겠어요.
https://javadoc.io/doc/org.mockito/mockito-core/latest/org/mockito/Mockito.html#19

```suggestion
        // given
        given(menuGroupDao.existsById(anyLong()))
```